boards/arm/nucleo_wb55rg/nucleo_wb55rg.yaml

```diff
@@ -10,10 +10,12 @@ ram: 96
 flash: 1024
 supported:
   - gpio
-  - arduino_i2c
   - i2c
   - counter
   - spi
   - pwm
   - adc
   - watchdog
+  - arduino_serial
```

Comment:
I know this was my request but `arduino_serial` doesn't exists, this should actually be `arduino_gpio`. My bad.
Can you do the change?